public/components/notebook/notebook_name_modal.tsx

```diff
@@ -16,22 +16,54 @@ import {
   EuiModalHeaderTitle,
 } from '@elastic/eui';
 import React, { useState, useCallback } from 'react';
+import { useCore } from '../../contexts/core_context';
+import { toMountPoint } from '../../../../../src/plugins/opensearch_dashboards_react/public';
 
-interface Props {
+export interface NotebookNameModalProps {
   onClose: () => void;
   // SaveChat hook depends on context. Runtime modal component can't get context, so saveChat needs to be passed in.
   saveChat: (name: string) => void;
 }
 
-export const NotebookNameModal = ({ onClose, saveChat }: Props) => {
+export const NotebookNameModal = ({ onClose, saveChat }: NotebookNameModalProps) => {
+  const {
+    services: {
+      notifications: { toasts },
+    },
+  } = useCore();
   const [name, setName] = useState('');
   const [loading, setLoading] = useState(false);
 
   const onSubmit = useCallback(async () => {
     setLoading(true);
-    await saveChat(name);
+    try {
+      const notebookId = await saveChat(name);
+      const notebookLink = `./observability-notebooks#/${notebookId}?view=view_both`;
+
+      toasts.addSuccess({
+        text: toMountPoint(
+          <>
+            <p>
+              This conversation was saved as{' '}
+              <EuiLink href={notebookLink} target="_blank">
+                {name}
+              </EuiLink>
+              .
+            </p>
+          </>
+        ),
+      });
+    } catch (error) {
+      if (error.message === 'Not Found') {
```

Comment:
This may out of the scope of this PR, but I'm wondering if it's ok to assume `observability plugin` is not installed when error message is `Not Found`? Because `Not Found` sounds like a general error message.

Reply:
I think we can assume they are same because observability will register this route automatically if it is installed. OSD server will throw `Not Found` error if this route is not registered. I tried to import an error constant from OSD to avoid directly judge this string, but OSD doesn't export.